examples/popcount/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"teak"
+
+	// Import support for AXI memory access
+	aximemory "axi/memory"
+	axiprotocol "axi/protocol"
+	// Import the entire framework (including bundled verilog)
+	_ "sdaccel"
```

Comment:
Currently, I'm not sure what our compiler does with underscored inputs, but it does not yet have `init()` functionality.